abi/abi/HEVM.sol

```diff
@@ -208,6 +208,7 @@ serializeString(string,string,string)(string)
 serializeString(string,string,string[])(string)
 serializeBytes(string,string,bytes)(string)
 serializeBytes(string,string,bytes[])(string)
+keyExists(string,string)(bool)
```

Comment:
Do we like this name, or should it be `jsonKeyExists`? Thinking ahead to support for https://github.com/foundry-rs/foundry/issues/4808 where we'd have cheats to parse TOML also. TOML can always be converted to JSON, so I think the options are:
- Namespace with `jsonKeyExists` and `tomlKeyExists`
- Keep a single `keyExists` method and forge detects if the string is JSON or TOML and parses it accordingly

The latter feels like nicer UX so the user doesn't have to worry about the file format of the data

Reply:
Yep, I was going to rename to `jsonKeyExists` to keep namespaces separate, but actually I think `keyExists` with automatic detection leads to a nicer UX, and it shouldn't be too hard of an additional lift to implement.